deploy/charts/rook-ceph/values.yaml

```diff
@@ -113,6 +113,9 @@ csi:
   # sidecar with CSI provisioner pod, to enable set it to true.
   enableOMAPGenerator: false
 
+  # -- Set CephFS Kernel mount options to use https://docs.ceph.com/en/latest/man/8/mount.ceph/#options.
+  cephFSKernelMountOptions:
```

Comment:
How about adding a comment to this helm setting, and also the operator.yaml examples to be clear that `ms_mode=secure` is needed to enable encryption on the wire for cephfs volumes

Reply:
Also how about adding a comment to the ceph-cluster-crd.md under the `connections.encrypted` setting to point them to set this value if they are enabling the encryption.

Reply:
The description looks good, just a small suggestion in the doc.